src/Avalonia.Animation/AnimationInstance`1.cs

```diff
@@ -31,22 +31,25 @@ internal class AnimationInstance<T> : SingleSubscriberObservableBase<T>
         private TimeSpan _initialDelay;
         private TimeSpan _iterationDelay;
         private TimeSpan _duration;
-        private Easings.Easing _easeFunc;
-        private Action _onCompleteAction;
+        private Easings.Easing? _easeFunc;
+        private Action? _onCompleteAction;
         private Func<double, T, T> _interpolator;
-        private IDisposable _timerSub;
+        private IDisposable? _timerSub;
         private readonly IClock _baseClock;
-        private IClock _clock;
-        private EventHandler<AvaloniaPropertyChangedEventArgs> _propertyChangedDelegate;
+        private IClock? _clock;
+        private EventHandler<AvaloniaPropertyChangedEventArgs>? _propertyChangedDelegate;
 
-        public AnimationInstance(Animation animation, Animatable control, Animator<T> animator, IClock baseClock, Action OnComplete, Func<double, T, T> Interpolator)
+        public AnimationInstance(Animation animation, Animatable control, Animator<T> animator, IClock baseClock, Action? OnComplete, Func<double, T, T> Interpolator)
         {
             _animator = animator;
             _animation = animation;
             _targetControl = control;
             _onCompleteAction = OnComplete;
             _interpolator = Interpolator;
             _baseClock = baseClock;
+            _lastInterpValue = default!;
```

Comment:
Unfortunately I don't think there's really a better way to do this, other than use `Optional<T>` but I didn't want to make that change here.